src/Raven.Server/Documents/DatabasesLandlord.cs

```diff
@@ -1050,18 +1063,23 @@ private void LogUnloadFailureReason(StringSegment databaseName, string reason)
                 _logger.Operations($"Could not unload database '{databaseName}', reason: {reason}");
         }
 
-        public void RescheduleDatabaseWakeup(string name, long milliseconds, DateTime? wakeup = null)
+        public void RescheduleNextIdleDatabaseActivity(string databaseName, IdleDatabaseActivity idleDatabaseActivity)
```

Comment:
Can we have only one overload of `RescheduleNextIdleDatabaseActivity`?

Reply:
Done.